wpe/src/main/glue/browser/entrypoints.cpp

```diff
@@ -194,6 +199,34 @@ Java_com_wpe_wpe_BrowserGlue_deleteInputMethodContent(JNIEnv *env, jclass clazz,
     Browser::getInstance().deleteInputMethodContent(pageId, offset);
 }
 
+JNIEXPORT void wpe_android_launchProcess(uint64_t pid, int processType, int *fds) {
+    ALOGV("BrowserGlue wpe_android_launchProcess pid: %ld processType: %d", pid, processType);
+
+    JNIEnv *env = wpe::android::AttachCurrentThread();
+    jclass jClass = env->GetObjectClass(s_BrowserGlue_object);
+    jmethodID jMethodID = env->GetMethodID(jClass, "launchProcess", "(JI[I)V");
```

Comment:
As s_BrowserGlue_object is a weak reference, the java instance may have been garbage collected. Maybe we should take a normal NewGlobalRef or check here that the instance is still valid.

Reply:
Fixed